### DISCUSSION TITLE
Interpreting normalisation results

### DISCUSSION THREAD

**Discussion by SalemWerdyani** [PGScatalog/pgsc_calc#284] [Q&A] [answered]
Hi everyone, 
I am new to the PGS calculation and a bit confused of reading the PRS results listed in the output.txt.gz file, which value is considered to be the PRS for each sample, the SUM or AVG in case of running the analysis without ancestry normalization, or SUM, Z_MostSimilarPop, percentile_MostSimilarPop, Z_norm1, or Z_norm2 in case of running the analysis with ancestry normalization, or there are further steps need to be done to calculate the score from these results? Thanks in advance for the help.

**Comment by smlmbrt** [comment, marked as answer]
The `SUM` is always the PGS (weighted sum of allele dosages) calculated on the matched variants. The `SUM` may change when you run with/without ancestry adjustment because the variants overlapping your genotypes and the reference panel might not be identical. 

The other methods (`Z_MostSimilarPop`, `percentile_MostSimilarPop`, `Z_norm1`, or `Z_norm2`) are different ways for reporting a PGS with respect to genetic ancestry, you can see more information here: https://pgsc-calc.readthedocs.io/en/latest/explanation/geneticancestry.html.

**Reply by ashenfernando1** [reply]
Hello, 

I ran the pgsc_calc pipeline on two cohorts - A and B, where B is a subset of A. --run_ancestry was not utilized. I am seeing different `SUM` values for the same sample across the two studies, is that to be expected? 

Thanks.

**Reply by smlmbrt** [reply]
Depends on how small the cohorts are. If there are missing genotypes they will be filled in with imputed allele frequencies based on the cohort, this could introduce some changes in the scores. Are they correlated?

**Comment by SalemWerdyani** [comment]
Thank you Sam for the clarification.


*Salem Werdyani,PhD.*
*Human Genetics.*
